core/src/main/java/org/axonframework/commandhandling/gateway/AbstractCommandGateway.java

```diff
@@ -43,24 +45,18 @@
     private final List<MessageDispatchInterceptor<? super CommandMessage<?>>> dispatchInterceptors;
 
     /**
-     * Initialize the AbstractCommandGateway with given {@code commandBus}, {@code retryScheduler} and
-     * {@code commandDispatchInterceptors}.
+     * Instantiate a {@link AbstractCommandGateway} based on the fields contained in the {@link Builder}.
+     * <p>
+     * Will assert that the {@link CommandBus} is not {@code null} and will throw an {@link AxonConfigurationException}
+     * if this is the case.
```

Comment:
This last sentence conflicts a bit. "if this is the case."

"this" could relate to "is not {@code null}". 
So, it will throw an exception when this is not the case, I would say. Or not use "this" and say something like:


```
Will assert that the {@link CommandBus} is not {@code null} and will throw an {@link AxonConfigurationException}
when the {@link CommandBus} is {@code null}.
```

Reply:
Adjusted javadoc to increase clarity.